src/main/java/com/mdudzisz/ticketbookingapp/service/TimeInterval.java

```diff
@@ -0,0 +1,106 @@
+package com.mdudzisz.ticketbookingapp.service;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.sql.Timestamp;
+import java.util.*;
+
+import static javax.xml.bind.DatatypeConverter.parseDateTime;
+
+@Getter
+@EqualsAndHashCode
+public class TimeInterval {
+
+    private final static String QUERY_FROM_TAG = "from";
+    private final static String QUERY_TO_TAG = "to";
+
+    private final static int MINUTES_BEFORE_CLOSEST_SCREENING = 15;
+
+    private final Timestamp from;
+    private final Timestamp to;
+
+    public static TimeInterval fromQueryMap(Map<String, String> queryMap) {
```

Comment:
Logika parsowania parametrów zapytania http zdecydowanie nie pasuje mi do klasy TimeInterval, to można załatwić wyżej. 

No i nazwane parametry zamiast mapy stringów zdecydowanie poprawiły by czytelność tej klasy.